topics/digital-humanities/tutorials/open-refine-tutorial/tutorial.md

```diff
@@ -0,0 +1,473 @@
+>
+> 1. Create a new history for this tutorial and name it "Powerhouse Museum — OpenRefine"
+> 2. Import the file from [Zenodo]({{page.zenodo_link}}):
+>
+>    ```
+>    https://zenodo.org/records/17047254
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the dataset: "**Powerhouse Museum metadata**."
+> 4. Ensure that the datatype is "tsv". Otherwise, use convert datatype.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+{: .hands_on}
+
+# Use OpenRefine to explore and clean your dataset
+
+The users will familiarize themselves with the museum's metadata. In the next step, they will clean the metadata using various steps to enhance its quality and accessibility.
+
+## Start OpenRefine
+
+> <hands-on-title>Opening the dataset with OpenRefine</hands-on-title>
+>
+> 1. Open the {% tool [OpenRefine](interactive_tool_openrefine) %}: Working with messy data  
+>    - *"Input file in tabular format"*:  `openrefine-phm-collection.tsv`  
+>
+> 2. Click on "Run Tool".  
+>
+>    <p align="center">
+>      <img src="openrefine.png" alt="OpenRefine tool interface in Galaxy">
+>    </p>
+>
+> 3. After around 30 seconds, using the interactive tools section on the left panel, you can open OpenRefine by clicking on its name. Make sure to wait until you see the symbol with an arrow > pointing outside the box that allows you to start OpenRefine in a new tab.  
+>
+>    <p align="center">
+>      <img src="interactive_tools.png" alt="Open OpenRefine tool as an Interactive tool">
+>    </p>
+>
+> 4. Here, you can see the OpenRefine GUI. Click on `Open Project`.  
+>
+>    <p align="center">
+>      <img src="openrefine_interface.png" alt="Open OpenRefine interface">
+>    </p>
+>
+> 5. Click on `Galaxy file`. If the file does not appear, you may have started OpenRefine before it was fully loaded. Retry steps 3 and 4, and the file should be visible.
+>
+>    <p align="center">
+>      <img src="openrefine_open_project.png" alt="Open OpenRefine Open Project as an input">
+>    </p>
+>
+> 6. You can see the data loaded for you.  
+>
+>    <p align="center">
+>      <img src="openrefine_gui.png" alt="Open OpenRefine GUI">
+>    </p>
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. How many rows does this table have?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. 75809
+> >
+> {: .solution}
+{: .question}
+
+Great, now that the dataset is in OpenRefine, we can start cleaning it.
+
+## Remove blank rows
+
+> <hands-on-title>Removing the blank rows</hands-on-title>
+>
+> 1. Click on the triangle on the left of `Record ID`.  
+>
+>    <p align="center">
+>      <img src="sort.png" alt="Sort Record ID">
+>    </p>
+>
+> 2. Click on `Sort...`.  
+>
+> 3. Select `numbers` and click on `OK`.  
+>
+>    <p align="center">
+>      <img src="sort2.png" alt="Sort Record ID options">
+>    </p>
```

Comment:
please use markdown syntax instead of html. If you did it for the centering, you can achieve that in markdown as follows:

```suggestion
>    ![Sort Record ID options](sort2.png)
```

if you really feel very strongly about the centering, you can do `![Sort Record ID options](sort2.png){: style="align: center"}` but I would use as little custom styling as possible in tutorials (perhaps we can make centering the default across the GTN, then you dont have to set it yourself, just let me know)